tests/benchmark/test_cross_validation.py

```diff
@@ -0,0 +1,100 @@
+from typing import Dict, Union
+import numpy as np
+import pandas as pd
+import pytest
+
+from qolmat.benchmark import cross_validation
+from qolmat.imputations.imputers import ImputerRPCA
+from qolmat.benchmark.missing_patterns import EmpiricalHoleGenerator
+
+df_origin = pd.DataFrame({"col1": [0, np.nan, 2, 4, np.nan], "col2": [-1, np.nan, 0.5, 1, 1.5]})
+df_imputed = pd.DataFrame({"col1": [0, 1, 2, 3.5, 4], "col2": [-1.5, 0, 1.5, 2, 1.5]})
+df_mask = pd.DataFrame(
+    {"col1": [False, False, True, True, False], "col2": [True, False, True, True, False]}
+)
+df_corrupted = df_origin.copy()
+df_corrupted[df_mask] = np.nan
+
+imputer_rpca = ImputerRPCA(tau=2, random_state=42)
+dict_imputers_rpca = {"rpca": imputer_rpca}
+generator_holes = EmpiricalHoleGenerator(n_splits=1, ratio_masked=0.5)
+dict_config_opti = {
+    "rpca": {
+        "lam": {"min": 0.1, "max": 1, "type": "Real"},
+        "max_iter": {"min": 99, "max": 100, "type": "Integer"},
+        "norm": {"categories": ["L1", "L2"], "type": "Categorical"},
+    }
+}
+dict_config_opti_imputer = dict_config_opti.get("rpca", {})
+hyperparams_flat = {"lam": 0.93382, "max_iter": 100, "norm": "L1"}
+
+cv = cross_validation.CrossValidation(
+    imputer=imputer_rpca,
+    dict_config_opti_imputer=dict_config_opti_imputer,
+    hole_generator=generator_holes,
+)
+
+result_params_expected = {"lam": (0.1, 1), "max_iter": (99, 100), "norm": ("L1", "L2")}
+
+
+@pytest.mark.parametrize("dict_bounds", [dict_config_opti_imputer])
+@pytest.mark.parametrize("param", ["lam", "max_iter", "norm"])
+def test_benchmark_cross_validation_get_dimension(dict_bounds: Dict, param: str) -> None:
+    result = cross_validation.get_dimension(dict_bounds=dict_bounds[param], name_dimension=param)
+    result_expected = result_params_expected[param]
+    np.testing.assert_equal(result.bounds, result_expected)
+
+
+@pytest.mark.parametrize("dict_config_opti_imputer", [dict_config_opti_imputer])
+def test_benchmark_cross_validation_get_search_space(dict_config_opti_imputer: Dict) -> None:
+    list_result = cross_validation.get_search_space(dict_config_opti_imputer)
+    result_expected = [
+        result_params_expected["lam"],
+        result_params_expected["max_iter"],
+        result_params_expected["norm"],
+    ]
+    for i in range(3):
```

Comment:
for result, expected_bounds in zip(list_result, list_expected_bounds):
   assert result.bounds == expected_bounds